src/NServiceBus.RavenDB/Subscriptions/Subscription.cs

```diff
@@ -14,6 +13,8 @@ class Subscription
 // ReSharper disable once UnusedAutoPropertyAccessor.Global
         public MessageType MessageType { get; set; }
 
+        List<SubscriptionClient> subscribers;
```

Comment:
Most repositories that still follow the resharper formatting that we started as part of the dotsettings file explicitly put fields on the bottom (more or less also in this project the case). So why deviate from that? This seems personal preference type formatting which I think we should try to avoid or if it makes sense roll it out consistently.